version/3_1.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash -e
+
+echo "================= Installing Python 3.1 ==================="
+sudo apt-get install -y \
+  python3.1=3.1*	 \
+  python3-dev=3.4*
```

Comment:
need to install python-pip and python-virtualenv for all python3.x versions

Reply:
this also needs to be updated in the design and docs

Reply:
ok